src/App/VPSScan/Scan/ScotlandYard.hs

```diff
@@ -15,10 +15,18 @@ import Data.Text (Text)
 import Network.HTTP.Req
 import Prelude
 import App.Util (parseUri)
+import Data.Text.Encoding
+
+-- Prefix for Core's reverse proxy to SY
+coreProxyPrefix :: Url 'Https -> Url 'Https
+coreProxyPrefix baseurl = baseurl /: "api" /: "proxy" /: "scotland-yard"
+
+authHeader :: Text -> Option scheme
+authHeader apiKey = header "Authorization" (encodeUtf8 ("token " <> apiKey))
```

Comment:
```suggestion
authHeader apiKey = header "Authorization" (encodeUtf8 ("Bearer " <> apiKey))
```
As far as I know, we made up the "token XXXXX" pattern for the Authorization header, but some tools like Swagger won't allow it, so to get [swagger.io](https://swagger.io/) to work for the Core API we needed to add support for the Bearer scheme: https://github.com/fossas/FOSSA/pull/4450

Reply:
Oh, nice. Thanks.